Extension/src/LanguageServer/configurations.ts

```diff
@@ -97,7 +98,8 @@ export interface Configuration {
     browse?: Browse;
 }
 
-export interface DefaultPaths {
+export interface CompilerDefaults {
+    compilerPath: string;
```

Comment:
It seems weird to send this back to the TypeScript code when the setting has already been sent down to the language server.  I presume you do this to resolve variables and such.  Can we intercept the didChangeConfiguration message and resolve the variables before they are sent down to the language server the first time?

Reply:
It's sent back so that when a user generates a new c_cpp_properties, the guessed one is shown.